src/build_interpret.ml

```diff
@@ -26,7 +26,33 @@ module Static_deps = struct
     }
 end
 
-let static_deps t ~all_targets =
+type file_kind = Reg | Dir
+
+let inspect_path file_tree path =
+  match Path.drop_build_context path with
+  | None ->
+    if not (Path.exists path) then
+      None
+    else if Path.is_directory path then
+      Some Dir
+    else
+      Some Reg
+  | Some path ->
+    match File_tree.find_dir file_tree path with
+    | Some _ ->
+      (* diml: we should probably warn in this case *)
```

Comment:
Why is this a warning?

Reply:
nvm, this was a leftover comment. I removed it